build.gradle

```diff
@@ -9,8 +9,7 @@ plugins {
   id "org.sonarqube" version "2.8"
 }
 
-// TODO: Update group to end with 'admin' or 'trainee'.
-group = 'uk.nhs.hee.tis'
+group = 'uk.nhs.hee.tis.trainee.reference'
```

Comment:
Should be `uk.nhs.hee.tis.trainee`.
This is used the same way Maven's `groupId` is, see https://github.com/Health-Education-England/TIS-TCS/blob/master/pom.xml#L6